### PR TITLE
DNS

### LINKED ISSUE #1079: DNS verifier
# Aim

- it checks the DNS returns IP and its availability.

# Details

- DNS verifier is a script in python which uses dnspython.

@avinashkranjan I am a GSSoC'21 participant. Kindly allow me to work on this issue and contribute to this project!

### PR DESCRIPTION
**Pull request by sur16-cse**
# Description

A Script to verify the Domain Name System.
- This is useful to check for the availability of DNS.

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
-[x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] I have added the project meta data in the PR template.
- [x] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [x] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: DNS verifier

Folder: DNS verifier

Requirements:  requirements.txt

Script: main.py

Arguments: --dns <DNS>

Contributor: sur16-cse

Description: Script to verify the Domain Name System. The argument is needed as to check users DNS request check.

### CODE REVIEW

File: DNS verifier/readme.MD [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# DNS VERIFIER
+## Description
+A Script to verify the Domain Name System.
+- This is useful to check for availability of DNS. 
+
+### Language
```
Comment (kaustubhgupta) [inline DNS verifier/readme.MD:6]:
```suggestion
## Language
```

File: DNS verifier/readme.MD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [X] Python
+  
+### Instructions to run this application
```
Comment (kaustubhgupta) [inline DNS verifier/readme.MD:9]:
```suggestion
## Instructions to run this application
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
minor change:

**Comment by sur16-cse** [comment]
@avinashkranjan  made the changes in readme please review it.

**Review by kaustubhgupta — COMMENTED** [review]
we generally face issues while integrating argument based scripts into the master script. Can you refactor your code that accepts DNS name from user and then you display the verification information?

**Comment by sur16-cse** [comment]
Yup sure i will update it

**Comment by sur16-cse** [comment]
@avinashkranjan restructured the code in main.py and made changes in the readme.MD accordingly. Please review it.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by sur16-cse** [comment]
Thanks

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**